src/Config/ObjectModel/AuthProvider.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+public enum AuthProvider
+{
+    StaticWebApps,
+    AppService,
+    AzureAD,
+    Jwt
```

Comment:
Is JWT a synonym to AzureAD for now?

Reply:
I dont think so, but I am not sure. @abhishekkumams, Can you add some info here as to how these possible values came about while creating the issue?